docs-source/content/samples/simple/elastic-stack/weblogic-domain/_index.md

```diff
@@ -0,0 +1,340 @@
+---
+title: "WebLogic Domain"
+date: 2019-10-01T14:32:31-05:00
+weight: 2
+description: "WebLogic domains and operator's logs"
+---
+
+
+#### Overview
+This page describes to how to configure a WebLogic domain to use fluentd to send log information to Elasticsearch.
+
+The general mechanism of how this works is:
+
+* `fluentd` runs as a separate container in the admin and managed server pods
+* The log files reside on a volume that is shared between the `weblogic-server` and `fluentd` containers
+* `fluentd` tails the domain logs files and exports to Elasticsearch
+* A `ConfigMap` contains the filter and format rules for exporting log records
+
+For sample purposes, this document will assume a domain with the following attributes are being configured:
+
+* Domain name is `bobs-bookstore`
+* Kubernetes namespace is `bob`
+* Kubernetes secret is `bobs-bookstore-weblogic-credentials`
+* Elasticsearch configuration
+```text
```

Comment:
This text block is not appearing correctly. It might need a blank line after the bulleted item, Elasticsearch configuration